docs/azure/TOC.yml

```
@@ -47,6 +47,8 @@
       href: ./sdk/pagination.md
     - name: Configure a proxy server
       href: ./sdk/azure-sdk-configure-proxy.md
+    - name: Resource Management
```

Prefer sentence casing for titles.

```suggestion
    - name: Resource management
```